specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -51,6 +51,555 @@
+        }
+      ],
+      "properties": {
+        "note": {
+          "description": "Router Job Note",
+          "type": "string"
+        },
+        "dispositionCode": {
+          "description": "Router Job Disposition Code",
+          "type": "string"
+        }
+      }
+    },
+    "AcsRouterJobClassificationFailedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobClassificationFailed event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventBaseProperties"
+        }
+      ],
+      "properties": {
+        "classificationPolicyId": {
+          "description": "Router Job Classification Policy Id",
+          "type": "string"
+        },
+        "errors": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterCommunicationError"
+          }
+        }
+      }
+    },
+    "AcsRouterJobClassifiedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobClassified event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventBaseProperties"
+        }
+      ],
+      "properties": {
+        "queueInfo": {
+          "description": "Router Job Queue Info",
+          "$ref": "#/definitions/AcsRouterQueueInfo"
+        },
+        "classificationPolicyId": {
+          "description": "Router Job Classification Policy Id",
+          "type": "string"
+        },
+        "priority": {
+          "description": "Router Job Priority",
+          "type": "integer",
+          "format": "int32"
+        },
+        "attachedWorkerSelectors": {
+          "description": "Router Job Attached Worker Selector",
+          "type": "array",
+          "format": {
+            "$ref": "#/definitions/AcsRouterWorkerSelector"
+          }
+        }
+      }
+    },
+    "AcsRouterJobClosedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobClosed event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventBaseProperties"
+        }
+      ],
+      "properties": {
+        "assignmentId": {
+          "description": "Router Job Closed Assignment Id",
+          "type": "string"
+        },
+        "workerId": {
+          "description": "Router Job Closed Worker Id",
+          "type": "string"
+        },
+        "dispositionCode": {
+          "description": "Router Job Closed Disposition Code",
+          "type": "string"
+        }
+      }
+    },
+    "AcsRouterJobCompletedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobCompleted event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventBaseProperties"
+        }
+      ],
+      "properties": {
+        "assignmentId": {
```

Comment:
This question applies to all string properties ending in `Id`:
Is this a GUID? Is there a corresponding API surface in the client libraries for ACS?